mmv1/third_party/terraform/tests/resource_workstations_workstation_config_test.go.erb

```diff
@@ -0,0 +1,94 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccWorkstationsWorkstationConfig_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
+		CheckDestroy: testAccCheckWorkstationsWorkstationConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWorkstationsWorkstationConfig_WorkstationConfigBasicExample(context),
```

Comment:
```suggestion
				Config: testAccWorkstationsWorkstationConfig_workstationConfigBasicExample(context),
```

I think this might be a casing issue? https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-7017-old..auto-pr-7017#diff-594d28d38e532db3d2f0d1711c97ce2b2d75144a43731f3d58a003b91b952ae9R51

Reply:
Sharp riley!

Reply:
I'm definitely juggling one to many things - I searched the referenced function and saw it wasn't present in the downstream and assumed - my bad